crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -239,9 +240,7 @@ pub struct AuthorityEpochTables {
 /// Parameters of the epoch fixed at epoch start.
 #[derive(Default, Serialize, Deserialize, Debug, Eq, PartialEq)]
 pub struct EpochStartConfiguration {
-    pub epoch_id: EpochId,
-    pub epoch_start_timestamp_ms: CheckpointTimestamp,
-    // Current epoch committee can eventually move here too, though right now it is served from a different table
+    pub system_state: SuiSystemState,
```

Comment:
nit: There are fields in `SuiSystemState` that will change over the course of the epoch.  IIUC, those changes will not be reflected in this field.  This is fine, but is there a risk that someone might not notice the distinction?  Couple of thoughts on mitigating that:

- Only use the system state object for initialisation, and mine out the fields that won't change.
- Rename this field to make it clear that this is the `initial_system_state` or something?

Reply:
Well so to be clear the object that stores this field is called Epoch**Start**Configuration